Gopkg.toml

```diff
@@ -11,7 +11,7 @@ required = [
 
 [[constraint]]
   name = "github.com/percona/pmm"
-  branch = "PMM-2.0"
+  branch = "PMM-5727"
```

Comment:
Please don't forget to update it once it is merged

Reply:
@percona-csalguero ^ that might be the cause of the problem. Please create a new PR that revert the revert commit, and then fixes this

Reply:
/cc @BupycHuk @askomorokhov

Reply:
No, anyway we shouldn't merge this PR, it changes logic.

Reply:
It is already merged and reverted :) I'm not saying we should merge a new PR – but we must create it

Reply:
I answered to first sentence about the cause of the problem. I agree with everything else.